src/poprox_recommender/domain/click_history.py

```diff
@@ -6,4 +6,4 @@
 
 class ClickHistory(BaseModel):
     account_id: UUID = None
-    article_ids: List[UUID]
+    article_id: List[UUID]
```

Comment:
Could we keep this as `article_ids`? I'm trying to distinguish between single values and list values with pluralization across the various domain concept classes